src/main/java/fi/aalto/cs/intellij/common/MalformedCourseConfigurationFileException.java

```diff
@@ -0,0 +1,21 @@
+package fi.aalto.cs.intellij.common;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MalformedCourseConfigurationFileException extends Exception {
+  @NotNull
+  private final String pathToConfigurationFile;
+
+  public MalformedCourseConfigurationFileException(@NotNull String pathToConfigurationFile,
+                                                   @NotNull String message,
+                                                   @Nullable Throwable cause) {
```

Comment:
this approach with each parameter on a new line reads very nice, I think I'll apply it to own code!